src/js/controller/FramesListController.js

```diff
@@ -8,7 +8,11 @@
     NEW_FRAME : 'newframe'
   };
 
-  ns.FramesListController = function (piskelController, container) {
+  var i18n = '';
+
+  ns.FramesListController = function (piskelController, container, i18nConfig) {
+    i18n = i18nConfig;
```

Comment:
What is this doing?

Reply:
I noticed that passing down the i18n object in the function was getting too involved I decided to make the i18n a global variable that  can be easier to access for the FrameListController

Reply:
First thought is that `var i18n = '';` is initializing i18n to the wrong type. It should either be `undefined` or an empty object `{}` since i18n should always be pointing to an object.

Second thought is that this is initializing an object and should be set to the object properties like the other parameters are. Remove the global `var i18n` and instead have something like `this.i18n = i18n`. But maybe I'm missing something and this won't work?

Reply:
Okay I will give this.i18n a try since this looks like a better solution!